code/datums/diseases/advance/symptoms/braindamage.dm

```diff
@@ -0,0 +1,52 @@
+/datum/symptom/braindamage
+	name = "Neural Decay"
+	desc = "Causes the host's brain cells to naturally die off, causing severe brain damage."
+	stealth = 1
+	resistance = -2
+	stage_speed = -3
+	transmittable = -1
+	level = 8
+	severity = 3
+	symptom_delay_min = 15
+	symptom_delay_max = 30
+	var/fastdamage = FALSE
+	var/moretrauma = FALSE
+	threshold_desc = "<b>Resistance 10:</b> Causes the host's brain cells to die off quicker.<br>\
+					  <b>Speed 9:</b> Host's brain develops even more traumas than normal."
+
+/datum/symptom/braindamage/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["transmission"] >= 12)
+		fastdamage = TRUE
+	if(A.properties["speed"] >= 9)
+		moretrauma = TRUE
+
+/datum/symptom/braindamage/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	switch(A.stage)
+		if(1)
+			if(prob(10))
+				to_chat(M, "<span class='notice'>Your head feels strange...</span>")
+		if(2, 3)
+			if(prob(10))
+				to_chat(M, "<span class='danger'>Your brain begins hurting...</span>")
+		if(4, 5)
+			if(fastdamage)
+				if(prob(10))
+					M.adjustBrainLoss(5, 90)
+					to_chat(M, "<span class='danger'>Your brain hurts immensely!</span>")
+			else
+				if(prob(3))
+					M.adjustBrainLoss(5, 90)
+					to_chat(M, "<span class='danger'>Your head hurts immensely!</span>")
+			if(moretrauma && A.stage == 5)
+				givetrauma(A, 1)
+
+/datum/symptom/braindamage/proc/givetrauma(datum/disease/advance/A, chance)
```

Comment:
Making this a proc seems unnecessary, you only use it once.

Reply:
See the give_clown_shoes proc. You can't give a `/mob/living` a trauma, but you can give a `/mob/living/carbon/human` one.